client/src/panels/panel_declarations/results/result_drilldown/result_drilldown.js

```diff
@@ -189,7 +189,7 @@ const get_col_defs = createSelector(
 class SingleSubjExplorer extends React.Component {
   constructor() {
     super();
-    this.state = { query: "" };
+    this.state = { query: "", is_expanded: false };
```

Comment:
Having the expand/collapse button be stateful is a can of worms. Adds complexity to the code and the site, and actually creates a usability deadzone IMO.

For site complexity, it's an issue of the number of possible interacting states. Depending on your path, you can end up with the same drilldown state but either button state. That's tricky for users and is also the sort of interplay that can leave more room for future bugs.

For the "usability deadzone", imagine a user clicks the button to expand everything, then starts collapsing things as they read, but later decides they want every thing expanded again. They can go back to the button they remember clicking, but it's been replaced by a collapse all. There's a lot of branches their behaviour could take at this point, but none of them are ideal (best case they figure out to click the button twice, but that's a pain even if the performance can be improved).

A possible fix is to sync the buttons state to the drilldowns state (e.g. if ANY nodes are still collapsed, the button acts as expand all; only collapses all if all are open). Pros: would at least make the state consistent (less room for state related bugs, better for users who can intuit the hidden rules). Cons: more code (and thus more room for bugs in general) and still a overly complex UX. 

My suggested fix is to split the behaviour in to two stateless, always present, buttons. One to expand all, one to collapse all. Sure, more controls on screen, but much less complexity for both users and developers (and the users always have a clear path to either option).